research-hub-web/src/app/app.component.html

```diff
@@ -98,7 +98,7 @@
                 </button>
 
                 <mat-menu #userMenu="matMenu" [overlapTrigger]="false">
-                  <div *ngIf="!showLoginBtn" style="margin-right: 8px; margin-left: 8px;">
+                  <div style="margin-right: 8px; margin-left: 8px;">
```

Comment:
:neckbeard: I'm a stickler for giving things styles sorry, even for basic things like this. 
Move the hardcoded CSS to an id/class.